client/src/main/com/sinch/sdk/domains/voice/adapters/converters/SAVMLActionDtoConverter.java

```diff
@@ -105,15 +106,15 @@ private static SvamlActionConnectPstnDto convertAction(ActionConnectPstn client)
     }
     SvamlActionConnectPstnDto dto = new SvamlActionConnectPstnDto();
     dto.setName(SvamlActionConnectPstnDto.NameEnum.CONNECTPSTN.getValue());
-    dto.setNumber(E164PhoneNumberDtoConverter.convert(client.getNumber()));
-    dto.setLocale(client.getLocale());
-    dto.setMaxDuration(client.getMaxDuration());
-    dto.setDialTimeout(client.getDialTimeout());
-    dto.setCli(client.getCli());
-    dto.setSuppressCallbacks(client.getSuppressCallbacks());
-    dto.setDtmf(DualToneMultiFrequencyDtoConverter.convert(client.getDtmf()));
-    dto.setIndications(EnumDynamicConverter.convert(client.getIndications()));
-    dto.setAmd(convert(client.getAnsweringMachineDetectionEnabled()));
+    client.getNumber().ifPresent(f -> dto.setNumber(E164PhoneNumberDtoConverter.convert(f)));
+    client.getLocale().ifPresent(dto::setLocale);
+    client.getMaxDuration().ifPresent(dto::setMaxDuration);
+    client.getDialTimeout().ifPresent(dto::setDialTimeout);
+    client.getCli().ifPresent(dto::setCli);
+    client.getSuppressCallbacks().ifPresent(dto::setSuppressCallbacks);
+    client.getDtmf().ifPresent(f -> dto.setDtmf(DualToneMultiFrequencyDtoConverter.convert(f)));
+    client.getIndications().ifPresent(f -> dto.setIndications(EnumDynamicConverter.convert(f)));
+    client.getAnsweringMachineDetectionEnabled().ifPresent(f -> dto.setAmd(convert(f)));
```

Comment:
Nitpicking: the setter is an acronym whereas the getter uses the full words

Reply:
Yes here the getter is the client facing function name and public but the setter is for the dto/generated and not public.
`setAmd` is coming from OAS file defining field `amd`; then generator created the `setAmd` setter
For client facing I choose a more informative naming `AnsweringMachineDetectionEnabled`
1. Name is wrong; it should be `AnsweringMachineDetection` without `Enabled`suffix
2. Class is not consistent with itself: other field `dtmf`  do not have more informative name. It should have been getDualToneMultiFrequency/setDualToneMultiFrequency
Working onto it

Reply:
Fixed

Reply:
Thanks for the clarification client vs DTO!